graphql.schema.json

```diff
@@ -32151,6 +32364,12 @@
         "inputFields": null,
         "interfaces": null,
         "enumValues": [
+          {
```

Comment:
Some changes here are from the base branch

Reply:
This should be updated in the base branch now